### PR TITLE
fix #1897

### LINKED ISSUE #1897: main site link in the main site is broken
It takes you to a page that doesn't exist.

### PR DESCRIPTION
**Pull request by antgonza**
Once merged we will need to update the config files (for each site) in the qiita main system deploy.

### CODE REVIEW

File: qiita_pet/support_files/config_portal.cfg [hunk 1]
```diff
@@ -32,7 +32,11 @@ TEXT = <p align="justify">
   </p>
   <p align="justify">
   For more information about how to use Qiita visit the
-  <a href=”static/doc/html/index.html”>documentation</a>
+  <a href=”static/doc/html/index.html”>documentation</a>.
+  </p>
+  <p align="justify">
+  Note that you should be log into the system to access to any studies and
```
Comment (ElDeveloper) [inline qiita_pet/support_files/config_portal.cfg]: log -> logged

Comment (ElDeveloper) [inline qiita_pet/support_files/config_portal.cfg]: to any -> the

**Comment by josenavas** [comment]
AFAIK we will not need to change anything. This file is the one used by default and the portals have a different one that does not contain this information, so I think once we deploy the change will be automatically done.

**Comment by josenavas** [comment]
👍

**Comment by josenavas** [comment]
I don't think we need for the tests to pass on this since it is only changing documentation that does not affect the tests. @ElDeveloper feel free to merge once your comments are addressed.

**Pull request merged**